README.md

```diff
@@ -58,10 +58,7 @@ WERROR=false make
 Enable additional sanitizers to catch hard-to-find bugs, for example:
```

Comment:
Perhaps this could say something like:

> Run tests locally with compile flags enabled to catch hard-to-find bugs (see [this PR](https://github.com/mapbox/node-cpp-skel/pull/85) for more info):
```
make sanitize
```

Reply:
👍   How does e57770a look?

Reply:
Sweet, added a comment inline. Also, I pulled the branch down to run `make sanitize` locally and got the following error. Looks like leak flag isnt supported by `x86_64-apple-darwin16.5.0`:

<details>

clang-4.0: error: unsupported option '-fsanitize=leak' for target 'x86_64-apple-darwin16.5.0'
make[2]: *** [Debug/obj.target/module/src/module.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/carol/.nvm/versions/node/v4.4.5/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:285:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 16.5.0
gyp ERR! command "/Users/carol/.nvm/versions/node/v4.4.5/bin/node" "/Users/carol/.nvm/versions/node/v4.4.5/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--error_on_warnings=true" "--loglevel=error" "--debug" "--module=/Users/carol/dev/node-cpp-skel/lib/binding/module.node" "--module_name=module" "--module_path=/Users/carol/dev/node-cpp-skel/lib/binding"
gyp ERR! cwd /Users/carol/dev/node-cpp-skel
gyp ERR! node -v v4.4.5
gyp ERR! node-gyp -v v3.6.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/Users/carol/.nvm/versions/node/v4.4.5/bin/node /Users/carol/.nvm/versions/node/v4.4.5/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --error_on_warnings=true --loglevel=error --debug --module=/Users/carol/dev/node-cpp-skel/lib/binding/module.node --module_name=module --module_path=/Users/carol/dev/node-cpp-skel/lib/binding' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/carol/dev/node-cpp-skel/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:87:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:827:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
node-pre-gyp ERR! System Darwin 16.5.0
node-pre-gyp ERR! command "/Users/carol/.nvm/versions/node/v4.4.5/bin/node" "/Users/carol/dev/node-cpp-skel/node_modules/.bin/node-pre-gyp" "configure" "build" "--error_on_warnings=true" "--loglevel=error" "--debug"
node-pre-gyp ERR! cwd /Users/carol/dev/node-cpp-skel
node-pre-gyp ERR! node -v v4.4.5
node-pre-gyp ERR! node-pre-gyp -v v0.6.36
node-pre-gyp ERR! not ok 
Failed to execute '/Users/carol/.nvm/versions/node/v4.4.5/bin/node /Users/carol/.nvm/versions/node/v4.4.5/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --error_on_warnings=true --loglevel=error --debug --module=/Users/carol/dev/node-cpp-skel/lib/binding/module.node --module_name=module --module_path=/Users/carol/dev/node-cpp-skel/lib/binding' (1)
make[1]: *** [debug] Error 1
make: *** [sanitize] Error 2

</details>

Reply:
`make distclean` cleared it up. Local sanitize looks 👍